Armadillo/src/main/java/com/scribd/armadillo/encryption/SecureStorage.kt

```diff
@@ -73,28 +78,30 @@ internal class ArmadilloSecureStorage @Inject constructor(
     override fun saveDrmDownload(context: Context, id: String, drmDownload: DrmDownload) {
         val alias = getDrmDownloadAlias(id, drmDownload.drmType)
         val value = Base64.encodeToString(Json.encodeToString(drmDownload).toByteArray(StandardCharsets.UTF_8), Base64.NO_WRAP)
-        secureDrmStorage.edit().putString(alias, value).apply()
+        secureDrmStorage?.edit()?.putString(alias, value)?.apply()
```

Comment:
Should we try to put it in legacy if secure doesn't exist?

Reply:
That would be failing insecure.